pytests/tests/test_pyclasses.py

```diff
@@ -71,9 +71,16 @@ def __new__(cls):
 
 
 @pytest.mark.parametrize(
-    "cls", [pyclasses.ClassWithoutConstructor, ClassWithoutConstructor]
+    "cls, exc_message",
+    [
+        (
+            pyclasses.ClassWithoutConstructor,
+            "cannot create 'builtins.ClassWithoutConstructor' instances",
+        ),
+        (ClassWithoutConstructor, "No constructor defined for ClassWithoutConstructor"),
```

Comment:
This type is defined just above; we can change the message to match the expected Rust one to simplify the test. (Then there's just the differences across versions.)